env/python/MANIFEST.in

```diff
@@ -9,5 +9,7 @@ include srwpy/*.so
 # If including data files in the package, add them like:
 # include path/to/data_file
 
+graft srwpy/examples/data_example_*
```

Comment:
This file - `MANIFEST.in` is used for the source distribution. To include the data files into the wheel packages, you'd add it via `setup.py`, such as in this example:
https://github.com/NSLS-II/scientific-python-cookiecutter/blob/af48f308a0a81f5ee7ae88abba5f393408bdd262/%7B%7B%20cookiecutter.repo_name%20%7D%7D/setup.py#L58-L64